uidl/structure.md

```diff
@@ -164,7 +164,7 @@ This is how you define an image `element`:
   "content": {
     "elementType": "image",
     "attrs": {
-      "src": "path/to/avatar.jpg"
+      "url": "path/to/avatar.jpg"
```

Comment:
This was the one instance of "src" over "url" that I felt needed to be updated to match the platform-agnostic decision.